src/sentry/sentry_metrics/querying/metadata/metric_spans.py

```diff
@@ -74,16 +74,16 @@ class Span:
 
 
 @dataclass(frozen=True)
-class MetricSpans:
+class MetricCorrelations:
     metric_mri: str
-    spans: Sequence[Span]
+    segments: Sequence[Segment]
 
     def __hash__(self):
         # For the serializer we need to implement a hashing function that uniquely identifies a metric.
         return hash(self.metric_mri)
 
 
-class SpansSource(ABC):
+class CorrelationsSource(ABC):
```

Comment:
Renamed to be more generic since we are still unsure whether or not we want to return spans, segments or traces.